apps/passport-server/migrations/39_semaphore_id.sql

```diff
@@ -0,0 +1,13 @@
+-- Step 1: Add semaphore_id column
+ALTER TABLE telegram_bot_conversations
+ADD column semaphore_id VARCHAR NOT NULL;
+
+-- Step 2: Add foreign key constraint on users commitment
+ALTER TABLE telegram_bot_conversations
+ADD CONSTRAINT fk_semaphore_commitment
+FOREIGN KEY (semaphore_id) REFERENCES users(commitment) ON UPDATE CASCADE;
```

Comment:
Are we sure that we want to cascade updates?

Reply:
Yes, because if we don't, when a user reset their semaphore id, the telegram_bot_conversations table has an incorrect mapping between user and id. Thus, a user's second account could use their new semaphore id to join the group.